### DISCUSSION TITLE
website without port / only path

### DISCUSSION THREAD

**Discussion by schmidtas** [NginxProxyManager/nginx-proxy-manager#3224] [General]
I already use NginxProxyManager for 4 webpages without any problems. Now I would like to run our time tracking via the proxy. I can do whatever I want, I can't manage it. The frontend is made available via port 80. But according to the manufacturer, the frontend cannot be opened via the port. Only a provided path works. Now I have the problem that Nginx necessarily expects a port. Has anyone ever had a problem like this? And if so, is there a solution for this?

**Comment by maurxce** [comment]
So you're trying to host a static website?
For that you can just create a proxy host that listens to `127.0.0.1:80` and use the `Advanced`-Configuration tab

Here's an example:
```
location / {
  root /data/website;
}
```

Then you just put your website into /data/website and you're good to go.
Again, just an example, your setup could differ from mine
